Doc/library/math.rst

```diff
@@ -206,6 +206,21 @@ Number-theoretic and representation functions
    :meth:`x.__trunc__() <object.__trunc__>`.
 
 
+.. function:: comb(n, k)
+
+   Return the binomial coefficient indexed by the pair of integers n >= k >= 0.
+
+   It is the coefficient of kth term in polynomial expansion of the expression
+   (1 + x)^n. It is also termed as the number of ways to choose an unordered
```

Comment:
Using the phrase `it is also termed as` isn't really as English idiom.  It reads better to me as `it is also known as`.

Reply:
Yeah, known sounds more correct. Fixed it in docstring and regenerated the clinic code.